scripts/bootstrap

```diff
@@ -5,7 +5,7 @@ cd $(dirname $0)/..
 
 apt-get update
 apt-get install -y python-pip python-dev tcpdump wget libfreetype6 fontconfig 
-pip install --upgrade pip==6.0.3 tox==1.8.1 virtualenv==12.0.4
+pip install --upgrade pip==6.0.3 tox==1.8.1 virtualenv==12.0.4 websocket=0.2.1
 
```

Comment:
We need websocket-client. Lets add it to requirements.txt and let tox handle it.